modules/k8s-cluster/main.tf

```diff
@@ -40,6 +40,14 @@ resource "aws_cloudwatch_log_group" "eks" {
   }
 }
 
+data "aws_ami" "node_ami" {
```

Comment:
does this take the region into account automatically? 

Given the werid release process for the EKS optimised AMIs, this seems a bit more opaque than grabbing the AMI id from the docs (ami-0f454b09349248e29)

Reply:
chatted with @Krenair ... sounds good :+1:

Reply:
You can check image name vs. ID in the EC2 console, under AMIs and with 'Public images' instead of 'Owned by me'